docker-compose.yml

```diff
@@ -111,38 +113,75 @@ services:
       - postgres:database.service.rasterfoundry.internal
 
   backsplash:
-    image: openjdk:8-jre
-    build:
-      context: app-backend
-      dockerfile: Dockerfile.backsplash
+    image: quay.io/azavea/openjdk-gdal:2.3.2-jdk8-slim
     depends_on:
       postgres:
         condition: service_healthy
     links:
       - postgres:database.service.rasterfoundry.internal
+      - memcached:tile-cache.service.rasterfoundry.internal
     env_file: .env
     environment:
       - RF_LOG_LEVEL=DEBUG
       - COURSIER_CACHE=/root/.coursier
+      - VSI_CACHE=TRUE
```

Comment:
Surfacing these here seems like a good idea. Your approach was to take the default values from the Scala configuration?

Reply:
These were already set in the `docker-compose.java.yml` file. I didn't add or remove any environment variables.